src/main/kotlin/de/amplimind/codingchallenge/config/SecurityConfig.kt

```diff
@@ -40,6 +41,18 @@ class SecurityConfig {
             .build()
     }
 
+    @Bean
+    fun cors(): WebMvcConfigurer {
```

Comment:
Und ich glaub um cors bei uns in Spring Security verwenden zu können braucht man eher so etwas hier in den neueren versionen

`@Bean
    fun cors() : CorsConfiguration {
        val cors = CorsConfiguration()
        cors.allowedOrigins = listOf("http://localhost:*")
        cors.allowedMethods = listOf("POST", "PUT", "DELETE")
        cors.allowedHeaders = listOf("*")
        return cors
    }`
    
    und dann oben in der `securityFilterChain` methode:
    
    kannst du cors mitteln `.cors { cors() }` hinzufügen

Reply:
stimmt, ich habs jtz angepasst anhand der offiziellen spring security doc (https://docs.spring.io/spring-security/reference/servlet/integrations/cors.html)